modules/weblog/metrics.go

```diff
@@ -130,3 +132,12 @@ func newReqCustomField(fields []customField) map[string]metrics.CounterVec {
 	}
 	return cf
 }
+
+// convert histogram options to microseconds (ms => us)
+func convHistOptionsToMicroseconds(histogram []float64) []float64 {
+	var buckets []float64
+	for _, value := range histogram {
+		buckets = append(buckets, float64(time.Duration(value)*time.Microsecond))
```

Comment:
I think we can just `* 1000`.

```suggestion
		buckets = append(buckets, value * 1000)
```

Reply:
it was like this, I thought maybe would be better to not hard code 1000.